lib/grantinee/engine/mysql.rb

```diff
@@ -44,7 +44,8 @@ def grant_permission!(data) # rubocop:disable Metrics/AbcSize
         fields   = data[:fields].map { |v| sanitize_column_name(v.to_s) }.join(', ')
 
         query = if data[:fields].empty?
-                  "GRANT #{kind} ON #{database}.#{table} TO '#{user}'@'#{host}';"
+                  type_helper = kind == "EXECUTE" ? " PROCEDURE " : ""
```

Comment:
Maybe it would be a little bit more readable if we wrapped ternary operators in ()? :-) Like:

`type_helper = (kind == "EXECUTE" ? " PROCEDURE " : "")`

Reply:
👍